cgp/utils.py

```diff
@@ -11,22 +11,39 @@
 
 
 def __check_cache_consistency(fn: str, func: Callable[..., float]) -> None:
+    """Retrieve an entry from the cache, execute the callable with the
+    cached arguments and check whether the return value matches the
```

Comment:
```suggestion
    cached arguments and check whether the return value matches the finite
```
?

Reply:
good point. i've added a warning to this docstring highlighting this.